tunnel/server.ts

```diff
@@ -0,0 +1,87 @@
+import { serve } from '@hono/node-server'
+import { Hono } from 'hono'
```

Comment:
I too like `hono` ! Nevertheless i don't know if we shouldn't use `fastify` like on the `ws` server, just for clarity. Wdyt @ericvicenti / @grabbou

Reply:
Yes, I haven't looked at the PR just yet (busy with other PRs), but generally speaking we should maintain whatever are our existing solutions, and if we change them - change everywhere for a good reason!